### ISSUE TITLE
Extract command error

### ISSUE THREAD

**Issue by andrewpmoore** [protomaps/go-pmtiles#52]
I've downloaded the sample 2.4gb level 10 zoom map and wanted to try and make one that only zooms to level 8
I'm trying to run the command to extract like this on a windows command line

`
pmtiles.exe extract --maxzoom=8 z10.pmtiles z8.pmtiles
`

But I just get the error
```
panic: extract <input> <output>

goroutine 1 [running]:
main. Main()
        /Users/bdon/workspace/protomaps/go-pmtiles/main.go:166 +0xc1e
```

Is there anything I'm doing wrong there, note that /Users/bdon/workspace is not my directory location, so not sure where that's coming from

**Comment by andrewpmoore** [comment]
Thanks @bdon, any idea how your go about getting a global level 0 to 8 map or modifying the 0 to 10 file supplied?

**Comment by bdon** [comment]
`pmtiles extract` as a experimental feature is being merged soon in #62 

This should allow `pmtiles extract input.pmtiles output.pmtiles --maxzoom 8` on a local file or remote storage bucket.

Closing this issue now, welcome to discuss in PR or #31
